create-issue/main.test.mjs

```diff
@@ -0,0 +1,102 @@
+import util from "node:util";
+
+test("create-issue-simple", async () => {
+  const mockPool = githubMockPool();
+
+  const token = "fake-token";
+  const title = "Issue title";
+  const body = "Issue body.\nLorem ipsum dolor sit amet.";
+  const labels = "label1,label2";
+  const assignees = "assignee1,assignee2";
+
+  const issueNumber = 12345;
+
+  mockInput("token", token);
+  mockInput("repository", GITHUB_REPOSITORY);
+  mockInput("title", title);
+  mockInput("body", body);
+  mockInput("labels", labels);
+  mockInput("assignees", assignees);
+
+  mockPool.intercept({
+    method: "POST",
+    path: `/repos/${GITHUB_REPOSITORY}/issues`,
+    headers: {
+      Authorization: `token ${token}`,
+    },
+    body: (htmlBody) => util.isDeepStrictEqual(JSON.parse(htmlBody), {
+      title,
+      body,
+      labels: labels.split(","),
+      assignees: assignees.split(","),
+    }),
+  }).defaultReplyHeaders({
+    "Content-Type": "application/json",
+  }).reply(200, {
+    number: issueNumber,
+  });
+
+  await loadMain();
+});
+
+test("create-issue-advanced", async () => {
+  const mockPool = githubMockPool();
```

Comment:
You can share some code by having a `describe` with nested `it`. See `post-comment` or `api-commit-and-push` as an example.